hack/ci/ci-run-offline-test.sh

```diff
@@ -243,7 +243,7 @@ retry 3 helm upgrade --install --force --wait --timeout 300 \
   --namespace ${NAMESPACE} \
   kubermatic-${BUILD_ID} charts/kubermatic/
 
-go build --tags "$KUBERMATIC_EDITION" github.com/kubermatic/kubermatic/cmd/conformance-tests
+go build --tags "$KUBERMATIC_EDITION" k8c.io/kubermatic/v2/cmd/conformance-tests
```

Comment:
```suggestion
go build --tags "$KUBERMATIC_EDITION" ./cmd/conformance-tests
```

How would you like the idea of replacing fully qualified package name (module name + package name) with relative path in FS? Of course that will require to make sure that the shell script which is launching this go build should take care, and that's highly debatable proposition :)

Reply:
Fixed.